debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/transforms/type/deserializer/ListTypeDeserializer.java

```diff
@@ -32,4 +34,16 @@ public SchemaBuilder getSchemaBuilder(AbstractType<?> abstractType) {
         Schema innerSchema = CassandraTypeDeserializer.getSchemaBuilder(elementsType).build();
         return SchemaBuilder.array(innerSchema);
     }
+
+    @Override
+    public Object deserialize(AbstractType<?> abstractType, ComplexColumnData ccd) {
+        List<ByteBuffer> bbList = ((ListType) abstractType).serializedValues(ccd.iterator());
+        AbstractType innerType = ((ListType) abstractType).getElementsType();
+        List<Object> deserializedList = new ArrayList<>();
+        for (int i = 0; i < bbList.size(); i++) {
```

Comment:
I'd suggest to use the for-each iteration style, unless you need the list index for other purposes:

    for(ByteBuffer bb : bbList) { ... }

Reply:
Updated!

Reply:
> Updated!



> I'd suggest to use the for-each iteration style, unless you need the list index for other purposes:
> 
> ```
> for(ByteBuffer bb : bbList) { ... }
> ```

Updated!